k8s/namespaces/fees-pay/ccpay-refunds-app/demo.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-refunds-api
+spec:
+  values:
+    java:
+      environment:
```

Comment:
You can add a dummy variable, which can be used to restart pods when required.

Reply:
dummy variable added